app/api/auth/validation.py

```diff
@@ -0,0 +1,55 @@
+from rest_framework import serializers
```

Comment:
I think its not really useful to separate out these functions from serializer, unless you want to reuse them.

Reply:
yes, it is reused in UpdateAccountSerializer